addons/account/account_invoice.py

```diff
@@ -739,7 +739,7 @@ def compute_invoice_totals(self, company_currency, ref, invoice_move_lines):
             if self.currency_id != company_currency:
                 currency = self.currency_id.with_context(date=self.date_invoice or fields.Date.context_today(self))
                 line['currency_id'] = currency.id
-                line['amount_currency'] = line['price']
+                line['amount_currency'] = currency.round(line['price'])
```

Comment:
@JulioSerna add a unit test where it fails without the patch and works with the patch please.

thanks.